cmd/mtaBuild.go

```diff
@@ -206,6 +206,11 @@ func runMtaBuild(config mtaBuildOptions,
 	call = append(call, "--source", getSourcePath(config))
 	call = append(call, "--target", getAbsPath(getMtarFileRoot(config)))
 
+	if config.CreateBOM {
+		log.Entry().Debugf("This feature is not included in the current version of the mta builder")
+		call = append(call, "--sbom-file-path", filepath.FromSlash("sbom-gen/bom-mta.xml"))
```

Comment:
Who will update this, when the feature is supported? What does the `--sbom-file-path` do?

Reply:
I have removed the log, --sbom-file-path is a flag from mbt cli which is used for providing output file path